src/perftest_resources.c

```diff
@@ -889,19 +889,23 @@ struct ibv_device* ctx_find_dev(const char *ib_devname)
 		return NULL;
 	}
 
-	if (!ib_devname) {
+	if (!*ib_devname) {
```

Comment:
mb it worth to add check for NULL before dereference?

Reply:
Agreed. :)